docs/concepts/glossary.md

```diff
@@ -240,6 +252,12 @@ The Relay is a means to establish connectivity between libp2p nodes (e.g., IPFS
 
 The Repository (Repo) is a directory where IPFS stores all its settings and internal data. It is created with the `ipfs init` command. [More about Repo](../how-to/command-line-quick-start.md#install-ipfs)
 
+### Root
+
+A root is a node in an [IPLD](#ipld) [graph](#graph) that by definition links to at least one [leaf](#leaf) or root together, roots may contain data too but that is optional.
+
+This is used to aggregate multiple chunks of a file together. For example if your file is 600MiB, which fits in 3 chunk (the block size of IPFS being 256MiB), you will create 3 leafs and then create a root linking all of them together. Then the CID of this last root is what IPFS shows you as the CID of the file.
```

Comment:
````suggestion
A root is a [node](#node) in a [graph](#graph) that links to at least one other node. In an IPLD graph, roots are used to aggregate multiple chunks of a file together. 

If you have a 600MiB file `A`, it can be split into 3 chunks `B`, `C`, and `D` since the block size of IPFS is 256MiB. The node `A` that links to each of these three chunks is the root. The CID of this root is what IPFS shows you as the CID of the file.

```
      A
      |
-------------
|     |     |
B     C     D
```
````